docs/src/pages/solid.mdx

```diff
@@ -106,11 +103,12 @@ import { generateComponents } from "@uploadthing/solid";
 
 import type { OurFileRouter } from "~/server/uploadthing";
 
-// URL is only needed for server side rendering
+// URL is only needed for server side rendering, or when your router
+// is deployed on a different path than `/api/uploadthing`
 const url = `http://localhost:${process.env.PORT ?? 3000}`;
 
 export const { UploadButton, UploadDropzone, Uploader } =
-  generateComponents<OurFileRouter>(url);
+  generateComponents<OurFileRouter>({ url });
```

Comment:
should we "take a chance" here that no-one is using this option 😅 

otherwise the Solid setup would drift from the React setup.

or should we still release this in a major bump?